packages/graphql/src/translate/connection/create-connection-and-params.ts

```diff
@@ -254,6 +254,13 @@ function createConnectionAndParams({
 
         const subqueryCypher = ["CALL {", subqueries.join("\nUNION\n"), "}"];
 
+        if (sortInput && sortInput.length) {
+            const sort = sortInput.map((s) =>
+                [...Object.entries(s.edge || []).map(([f, direction]) => `edge.${f} ${direction}`)].join(", ")
```

Comment:
I'm not sure why destructuring and creating a new array here. Wouldn't `Object.entries().map` already return an array? 
```suggestion
                Object.entries(s.edge || []).map(([f, direction]) => `edge.${f} ${direction}`).join(", ")
```

Reply:
Ahahaha, my mistake, maybe guilty of a little copy/paste from another part of the code where there is two array entries... 🙄